### ISSUE TITLE
please add test AFF4 images to the git repo

### ISSUE THREAD

**Issue by simsong** [google/aff4#15]
The git repo should contain some (small) test AFF4 images and the regression tests so check these images to make sure that the AFF4 implementation is working properly.

**Comment by blschatz** [comment]
Agreed. The python code contains some regression tests against the canonical images. It would be nice if we had similar in the C/C++ implementation.

**Comment by simsong** [comment]
Where are the canonical images?  Do the test scripts download them? They could be small enough so that they could be part of the git repo.

**Comment by blschatz** [comment]
Canonical images are in the ReferenceImages repository @ https://github.com/aff4/
